### ISSUE TITLE
SAML Federation in 2024.6.0 produces 405 Error

### ISSUE THREAD

**Issue by BlueKrypto** [goauthentik/authentik#10424]
**Describe the bug**
Authentik version 2024.6.0 federated to SAML Source by either Post or Redirect produces a 405. I have verified this issue on a system connecting two authentik systems and on with OneLogin as the source. Both were working fine before the upgrade from 2024.2.2

with our aws SAML application as source provider for Authentik but end up getting 405 error. The exact same setup works without any issue with earlier Authentik ver. 2024.2.3

**To Reproduce**

1. Stand up two authentik instances on 2024.6.0
2. Add provider and application on instance 1 as the Source
3. Add instance 1 to instance 2 as a federated source
4. Configure instance 2 to use instance 1 source as the authentication method
5. Attempt to log into instance 2
6. Receive 405 Error

**Expected behavior**
 Authorise user to login via SAML source.

**Logs**
``` 
   {
      "auth_via": "unauthenticated",
      "domain_url": "authentik.example.org",
      "event": "/source/saml/authentik1/acs/?SAMLResponse=...",
      "host": "authentik.example.org",
      "level": "info",
      "logger": "authentik.asgi",
      "method": "GET",
      "pid": 47,
      "remote": "10.42.0.1",
      "request_id": "e1b45a5aabb14b06a84866a609020886",
      "runtime": 18,
      "schema_name": "public",
      "scheme": "https",
      "status": 405,
      "timestamp": "2024-07-09T16:17:13.521460",
      "user": "",
      "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:126.0) Gecko/20100101 Firefox/126.0"
    }
```
**Version and Deployment (please complete the following information):**

-   authentik version: 2024.6.0
-   Deployment: helm

**Additional context**
This issues appears to be related to #9700

**Comment by BeryJu** [comment]
this is fixed by #10348 and will be released with 2024.6.1 later this week

**Comment by BlueKrypto** [comment]
> this is fixed by #10348 and will be released with 2024.6.1 later this week

@BeryJu I have just upgraded to the newest 2024.6.1

The `HTTP-Post` binding is working now, but the `redirect` binding is still production a 405 error:
![image](https://github.com/user-attachments/assets/22f1b6cd-f590-4deb-9550-5c30c4ec9c0d)

When I look at the metadata tab only the `HTTP-Post` method is included:

![image](https://github.com/user-attachments/assets/d62898e1-9865-4f1c-8b1e-3e39176f8970)

```xml
<md:EntityDescriptor entityID="authentik">
  <md:SPSSODescriptor protocolSupportEnumeration="urn:oasis:names:tc:SAML:2.0:protocol">
    <md:NameIDFormat>
      urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress
    </md:NameIDFormat>
    <md:NameIDFormat>
      urn:oasis:names:tc:SAML:2.0:nameid-format:persistent
    </md:NameIDFormat>
    <md:NameIDFormat>
      urn:oasis:names:tc:SAML:2.0:nameid-format:X509SubjectName
    </md:NameIDFormat>
    <md:NameIDFormat>
      urn:oasis:names:tc:SAML:2.0:nameid-format:WindowsDomainQualifiedName
    </md:NameIDFormat>
    <md:NameIDFormat>
      urn:oasis:names:tc:SAML:2.0:nameid-format:transient
    </md:NameIDFormat>
    <md:AssertionConsumerService isDefault="true" index="0" Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" Location="https://authentik2.bacustoms.org/source/saml/test/acs/"/>
  </md:SPSSODescriptor>
</md:EntityDescriptor>
```
